### ISSUE TITLE
fail to run preprocess.py

### ISSUE THREAD

**Issue by Bomingmiao** [jiamingzhang94/Unlearnable-Clusters#5]
I downloaded the pets datasets and split the train and test set according to the txt file in annotation. And when I run preprocess.py, there is an error below.
![image](https://github.com/jiamingzhang94/Unlearnable-Clusters/assets/110110792/037ce910-2db5-4228-bdc5-4d83c0a026b2)
What should I do to solve this problem? thx

**Comment by saigontrade88** [comment]
Hi,

I had a quick look at the 'preprocess.py" file and I tried to explain the error as follows.

The preprocess.py with the 'rename' option will first open the './config/preprocess.yaml' and read everything under the 'rename' group.

From the sample config file, './config/data.yaml' , the train and test data should be stored as 
- Training data: ./data/datasets/classification/pets/train/[labels], where labels are encoded as an integer starting from 0.
- Test data: ./data/datasets/classification/pets/test/[labels].

However, when line 130 is being executed for copying and pasting the test data, an error occurs. The error happens because './data/datasets/classification/pets/test**_raw**/0 does not exist. 

I know that I should resize the training and test images to have the same resolution like 128 x 128.
